server/src/services/UnsubscribeToken.ts

```diff
@@ -0,0 +1,32 @@
+import { sign } from 'jsonwebtoken';
+
+import { getConfig } from '../config';
+
+export enum unsubscribeType {
+  Chapter,
+  Event,
+}
```

Comment:
Now it crossed my mind whether shouldn't there be added values to these enums, to ensure they stay consistent in future (ie. in case of adding instance unsubscribing, or just in general).